keyboards/3.0/ja-t-k0-kana.xml

```diff
@@ -18,53 +18,53 @@
         <key id="Jko" to="き"/>
 	</keys>
 
-	<layerMaps form="hardware">
-		<layerMap modifier="none">
+	<layers form="hardware" hardware="jis">
+		<layer modifier="none">
 			<row keys="1 2 3 4 5 6 7 8 9 0 minus caret yen" /> <!-- delete -->
 			<row keys="Jta Jtu Ji Jsu Jka Jn Jna Jni Jra Jse rough smooth" />
 			<row keys="Jsa Jto Jsi Jha Jki Jku Jma Jno Jri Jre Jke Jmu" />
 			<row keys="shift Jtu Jti Jso Jhi Jko Jre Jyo Jne Jru Jme Jro shift" />
 			<row keys="eigo space kata" />
-		</layerMap>
+		</layer>
 
-		<layerMap modifier="shift">
+		<layer modifier="shift">
 			<row keys="1 2 3 4 5 6 7 8 9 0 degree plus" />
 			<row keys="A Z E R T Y U I O P umlaut pound" />
 			<row keys="Q S D F G H J K L M percent micro" />
 			<row keys="greater-than W X C V B N question-mark period slash section" />
 			<row keys="space" />
```

Comment:
This has only one key on the shift layer but the base layer has three keys.

Reply:
will fix